ddtrace/appsec/remoteconfiguration.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from ddtrace.constants import APPSEC_ENV
+from ddtrace.constants import RC_ENV
```

Comment:
I think the plan is to remove this as it would break those products that rely on RCM.

Reply:
remove the constants? or what do you mean? TY!

Reply:
Sorry, I meant the environment variable. From what I understood from RCM, there is no plan for exposing such a variable, as setting it to false would break all products that rely on RCM. Eventually, we would then want to get rid of this constant as it will be unused.